deploy/openshift/configmap.yaml

```diff
@@ -16,6 +16,9 @@ data:
         url: ${JAEGER_URL}
       grafana:
         url: ${GRAFANA_URL}
+    auth:
+      strategy: ${AUTH_STRATEGY}
+      secret: ${OAUTH_SECRET}
```

Comment:
this change to the configmap needs to be made upstream to istio and maistra.

Also, this breaks if you use the getLatestKiali (deploy-kiali-to-openshift.sh) because that script never sets AUTH_STRATEGY or OAUTH_SECRET (I assume a value of "" for these will break kiali).

Did we test the getLatestKiali installation method?